kafka-ui-react-app/src/components/Topics/Topic/Messages/Filters/Filters.tsx

```diff
@@ -577,6 +576,13 @@ const Filters: React.FC<FiltersProps> = ({
           </S.MetricsIcon>
           <span>{messagesConsumed} messages consumed</span>
         </S.Metric>
+        {filterApplyErrors ? (
+          <S.Metric title="Errors">
+            <span>{filterApplyErrors} errors</span>
+          </S.Metric>
+        ) : (
+          ''
```

Comment:
can we put `null` here instead of empty string, cause here React will try to render the empty string. 

or we can do it with short circuit checks such as 
```
{!!filterApplyErrors && (
          <S.Metric title="Errors">
            <span>{filterApplyErrors} errors</span>
          </S.Metric>
        )}
```